src/serial_device.h

```diff
@@ -170,6 +170,12 @@ enum class TDeviceConnectionState
     DISCONNECTED
 };
 
+enum class TDeviceSetupMode
+{
+    WITH_SETUP,
```

Comment:
```suggestion
    WITH_SETUP_IF_WAS_DISCONNECTED,
```
Иначе кажется, что мы всегда требуем записать регистры настройки.

Reply:
fixed